src/main/java/de/srendi/advancedperipherals/common/addons/computercraft/peripheral/RSBridgePeripheral.java

```diff
@@ -0,0 +1,920 @@
+            return notConnected();
+
+        Pair<ItemFilter, String> filter = ItemFilter.parse(arguments.optTable(0, Collections.emptyMap()));
+        if (filter.rightPresent())
+            return MethodResult.of(null, filter.getRight());
+
+        ItemFilter parsedFilter = filter.getLeft();
+
+        Set<Map<String, Object>> resourceProperties = RsApi.getCraftableItems(getNetwork(), parsedFilter);
+
+        return MethodResult.of(resourceProperties);
+    }
+
+    @Override
+    @LuaFunction(mainThread = true)
+    public final MethodResult listCraftableFluids(IArguments arguments) throws LuaException {
+        if (!isAvailable())
+            return notConnected();
+
+        Pair<FluidFilter, String> filter = FluidFilter.parse(arguments.optTable(0, Collections.emptyMap()));
+        if (filter.rightPresent())
+            return MethodResult.of(null, filter.getRight());
+
+        FluidFilter parsedFilter = filter.getLeft();
+
+        Set<Map<String, Object>> resourceProperties = RsApi.getCraftableFluids(getNetwork(), parsedFilter);
+
+        return MethodResult.of(resourceProperties);
+    }
+
+    @Override
+    @LuaFunction(mainThread = true)
+    public final MethodResult listCraftableChemicals(IArguments arguments) throws LuaException {
+        if (!isAvailable())
+            return notConnected();
+
+        if (!APAddons.mekanismLoaded || !APAddons.refinedStorageMekanismLoaded)
+            return MethodResult.of(Collections.emptyList());
+
+        Pair<ChemicalFilter, String> filter = ChemicalFilter.parse(arguments.optTable(0, Collections.emptyMap()));
+        if (filter.rightPresent())
+            return MethodResult.of(null, filter.getRight());
+
+        ChemicalFilter parsedFilter = filter.getLeft();
+
+        Set<Map<String, Object>> resourceProperties = RsApi.getCraftableChemicals(getNetwork(), parsedFilter);
+
+        return MethodResult.of(resourceProperties);
+    }
+
+    @Override
+    @LuaFunction(mainThread = true)
+    public final MethodResult listCells() {
+        if (!isAvailable())
+            return notConnected();
+
+        return MethodResult.of(RsApi.listCells(getNetwork()));
+    }
+
+    @Override
+    @LuaFunction(mainThread = true)
+    public final MethodResult listDrives() {
+        if (!isAvailable())
+            return notConnected();
+
+        return MethodResult.of(RsApi.listDrives(getNetwork()));
+    }
+
+    @Override
+    @LuaFunction(mainThread = true)
+    public final MethodResult importItem(IComputerAccess computer, IArguments arguments) throws LuaException {
+        if (!isAvailable())
+            return notConnected();
+
+        IItemHandler inventory = InventoryUtil.getHandlerFromDirection(arguments.getString(1), owner);
+
+        if (inventory == null)
+            inventory = InventoryUtil.getHandlerFromName(computer, arguments.getString(1));
+
+        if (inventory == null)
+            return MethodResult.of(0, StatusConstants.INVENTORY_NOT_FOUND.name());
```

Comment:
I'd prefer nested if in this case, same for below. It improves readability.

```suggestion
        if (inventory == null) {
            inventory = InventoryUtil.getHandlerFromName(computer, arguments.getString(1));
            if (inventory == null) {
                return MethodResult.of(0, StatusConstants.INVENTORY_NOT_FOUND.name());
            }
        }
```